userguide/system.rst

```diff
@@ -2024,52 +2028,52 @@ as the signing authority.
    |                         |                      |                                                                                                 |
    +-------------------------+----------------------+-------------------------------------------------------------------------------------------------+
-   | Locality                | string               | required; enter the location for the organization                                               |
+   | Locality                | string               | Enter the location for the organization.                                                        |
    |                         |                      |                                                                                                 |
    +-------------------------+----------------------+-------------------------------------------------------------------------------------------------+
-   | Organization            | string               | required; enter the name of the company or organization                                         |
+   | Organization            | string               | Enter the name of the company or organization.                                                  |
    |                         |                      |                                                                                                 |
    +-------------------------+----------------------+-------------------------------------------------------------------------------------------------+
-   | Email                   | string               | required; enter the email address for the person responsible for the CA                         |
+   | Email                   | string               | Enter the email address for the person responsible for the CA.                                  |
    |                         |                      |                                                                                                 |
    +-------------------------+----------------------+-------------------------------------------------------------------------------------------------+
-   | Common Name             | string               | required; enter the fully-qualified hostname (FQDN) of the system; the :guilabel:`Common Name`  |
-   |                         |                      | **must** be unique within a certificate chain                                                   |
+   | Common Name             | string               | Enter the fully-qualified hostname (FQDN) of the system. The :guilabel:`Common Name`            |
+   |                         |                      | **must** be unique within a certificate chain.                                                  |
    |                         |                      |                                                                                                 |
```

Comment:
s/must/Must/

Reply:
must is in the middle of a sentence here, are you sure you want to capitalize it?